packages/wdio-browserstack-service/src/service.ts

```diff
@@ -3,8 +3,12 @@ import got from 'got'
 import type { Services, Capabilities, Options, Frameworks } from '@wdio/types'
 import type { Browser, MultiRemoteBrowser } from 'webdriverio'
 
-import { getBrowserDescription, getBrowserCapabilities, isBrowserstackCapability, getParentSuiteName } from './util'
+import { getBrowserDescription, getBrowserCapabilities, isBrowserstackCapability, getParentSuiteName, isBrowserstackSession } from './util'
 import { BrowserstackConfig, MultiRemoteAction, SessionResponse } from './types'
+import type { Pickle, Feature } from '@cucumber/messages'
+import type { ITestCaseHookParameter } from '@cucumber/cucumber/lib/support_code_library_builder/types'
```

Comment:
Be aware that his means everyone even not using Cucumber will have to download this dependency. Better to mimic the types you need and just copy them in.